modules/apps/object/object-test/src/testFunctional/tests/macros/CreateObject.macro

```diff
@@ -0,0 +1,9 @@
+definition {
```

Comment:
I've just thought about a better name for this macro. Please, change the name to `ObjectAdmin.macro`

Reply:
Okay! @rodrigocunhaa